docs/content/themes.md

```diff
@@ -0,0 +1,84 @@
+title = "Bartholomew themes"
+date = "2022-07-20T22:44:01.300319091Z"
+
+[extra]
+
+---
+
+Bartholomew supports theming which allows for easy customization of the site along with the user defined tempaltes.
+
+## Adding a theme
+
+Once the initial site has been setup using the [quickstart section](/quickstart). Create a themes folder where you will be able to download different themes.
+```bash
+mkdir themes
+```
+
+Once the folder is created, different themes can be added as submodules to the folder which can then in turn be used to theme the site.
+
+```bash
+cd themes
+git submodule add <Source_to_the_theme>
+```
+
+Multiple themes can be added to the themes directory but only one of them will be active at a given time as described in the next section.
+
+## Configuring the site to use the theme
+
+To choose a theme for the website, the `theme` attribute in `config/site.toml` must be configured, where the value is the name of the theme as found in the `themes/` folder.
+
+```toml
+title = "Bartholomew Documentation"
+base_url = "http://localhost:3000"
+about = "The Micro-CMS for WebAssembly and Spin"
+theme = "<theme-directory>"
+
+[extra]
+copyright = "Fermyon"
+github = "https://github.com/fermyon/bartholomew"
+twitter = "https://twitter.com/fermyontech"
+ga_measurement_id = ""
+
+date_style = "%B %e, %Y"
+```
+One more step that needs to be done before themes are fully available to the site is to the change the static file server component in `spin.toml` configuration so that it provides the static assets of the selected theme. The convention of mounting the static assets of the themes before the user defined static assets is recommened.  
```

Comment:
```suggestion
One more step that needs to be done before themes are fully available to the site is to the change the static file server component in the `spin.toml` configuration so that it provides the static assets of the selected theme. The convention of mounting the static assets of the themes before the user-defined static assets is recommened.  
```